docs/docs/ledger/numscript/metadata/index.mdx

```diff
@@ -65,8 +65,22 @@ You can see the `metadata` field with the details of the information we added to
 
 ### What's going on here?
 
-Metadata is arbitrary data that you can attach to a transaction (or an account! But more on that later). This data is typed, and can be a string or a number. You can retrieve (or even add to) this metadata later using the HTTP API, for integration into your own bespoke workflows.
+Metadata is arbitrary data that you can attach to a transaction or an account. This data is typed, and can be a string or a number. You can retrieve (or even add to) this metadata later using the HTTP API, for integration into your own bespoke workflows.
+
+## Attaching account metadata
+
+You can also add metadata to an account during a transaction, like this:
```

Comment:
Since we're talking about it in the previous section, it seems odd to me to have the word 'also' in the beginning of this section, what do you think ?

Reply:
In this part, I'm using 'also' because the previous section talked about transaction metadata, not account metadata!